meta-mbl-distro/recipes-security/ecryptfs-utils/ecryptfs-utils_%.bbappend

```diff
@@ -2,17 +2,5 @@
 #
 # SPDX-License-Identifier: MIT
 
-
-SRC_URI += "\
-    file://ecryptfs.service \
-    file://ecryptfs-init.sh \
-    "
-
-FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
-
-do_install_append() {
- 
-    if ${@bb.utils.contains('DISTRO_FEATURES','systemd','true','false',d)}; then
-        install -D -m 0755 ${WORKDIR}/ecryptfs-init.sh ${D}${bindir}/ecryptfs-init.sh
-    fi
-}
+# Disable systemd integration.
```

Comment:
I believe that this bbappend needs to be moved to `meta-psa` since `ecryptfs-utils` is a dependency of the `psa-trusted-storage-linux` package and it requires the `ecryptfs.service` to be disabled.

Reply:
Moved.